src/components/AssetMatching/components/SectionHeader/SectionHeader.test.tsx

```diff
@@ -94,8 +94,8 @@ describe('Asset Matching - SectionHeader', () => {
     const primaryActions = getAllByTestId('primary_action');
 
     expect(primaryActions.length).toBe(2);
-    const uniqueSecondaryAction = await queryByTestId('icon-button-unique_secondary_action');
+    const menuBtn = await queryByTestId('icon-button-menu_btn');
```

Comment:
update test description :), `... and the menu button is not rendered.`

Reply:
I should rephrase my comment. You can change the description to -> `should render toolbox with 2 matching actions  and the menu button is not rendered.` or `should render toolbox with 2 matching actions without rendering the menu button.`